protocol/0062-SPAM-spam_protection.md

```diff
@@ -83,6 +83,17 @@ Further, each party is allowed to submit up to `n` transactions per epoch where
 
 **Note** `spam.protection.max.updatePartyProfile` must be an integer greater than or equal to `0` (and default to `5`).
 
+### Transaction Spam
+
+Network parameter: `spam.order.MinimalMarginQuantumMultiple` (between 0 and infinite)
+
+If the maintenance margin for a given transaction is smaller than the parameter `spam.order.MinimalMarginSizeQuantumMultiple`, then the transaction is pre-block rejected.
+
+If the market does not exist and thus the maintenance margin is not defined, the transaction is rejected.
```

Comment:
```suggestion
If the market does not exist and thus the maintenance margin is not defined, the transaction is rejected.

If floating point consensus hasn't delivered a risk factor yet use `rf = 1` for the spam check. 
```

Reply:
That can lead to inconsistencies between validators (some seeing the transaction before the consensus finished and some after). This is not a problem anymore, right @ze97286 ?

Reply:
They will all have an agreement on whether they have the true `rf` from the floating point consensus or not. So either none of them have it or all of them have it. In case none of them have it (because rf hasn't gone through consensus yet) use the value of `1`.

Reply:
But that won't cause any issues anymore, correct ?

Reply:
no